mqtt-to-blob/src/main.py

```diff
@@ -0,0 +1,80 @@
+from minio import Minio
+import logging
+import paho.mqtt.client as mqtt
+import numpy as np
+import os
+import base64
+import json
+import datetime
+import pytz
+import cv2
+import time
+
+#with open("./../config.json", "r") as config_file:
+#    config = json.load(config_file)
+
+broker_url = os.environ['BROKER_URL']
+broker_port = int(os.environ['BROKER_PORT'])
+minio_url = os.environ['MINIO_URL']
+minio_access_key = os.environ['MINIO_ACCESS_KEY']
+minio_secret = os.environ['MINIO_SECRET_KEY']
+bucket_name = os.environ['BUCKET_NAME']
+topic = os.environ['TOPIC']
+input_var = ""
+timezone = pytz.timezone('Europe/Vienna')
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Log exhaustion is not an issue here, therefore https://sonarcloud.io/project/security_hotspots?id=united-manufacturing-hub_united-manufacturing-hub&pullRequest=384&hotspots=AXrsY9dU6c7oe2nD05vG is safe.

This could be changed to use an remote logging service in the future.